modules/mcagar/src/main/ts/ephox/mcagar/api/bdd/TinyContentActions.ts

```diff
@@ -19,10 +19,18 @@ const keystroke = (editor: Editor, keyvalue: number, modifiers: Keyboard.KeyModi
 const type = (editor: Editor, content: string): void =>
   TypeText.typeContentAtSelection(TinyDom.document(editor), content);
 
+const trueClick = (editor: Editor): void =>
```

Comment:
These new functions need to be documented in the McAgar changelog.

- [ ] McAgar changelog entries please